File: backend/src/app/consts.py
import os
from dotenv import load_dotenv


load_dotenv()


BACKEND_HOST = os.environ.get("BACKEND_HOST", "0.0.0.0")
BACKEND_PORT = os.environ.get("BACKEND_PORT", 8000)
BACKEND_URL = f"http://{BACKEND_HOST}:{BACKEND_PORT}"

FRONTEND_HOST = os.environ.get("FRONTEND_HOST", "0.0.0.0")
FRONTEND_PORT = os.environ.get("FRONTEND_PORT", 3000)
FRONTEND_URL = f"http://{FRONTEND_HOST}:{FRONTEND_PORT}"

File: frontend/src/types/consts.ts
import { url } from "types/types"

const BACKEND_HOST = process.env.BACKEND_HOST
const BACKEND_PORT = process.env.BACKEND_PORT
export const BACKEND_URL = `http://${BACKEND_HOST}:${BACKEND_PORT}` as url

const FRONTEND_HOST = process.env.FRONTEND_HOST
const FRONTEND_PORT = process.env.FRONTEND_PORT
export const FRONTEND_URL = `http://${FRONTEND_HOST}:${FRONTEND_PORT}` as url

export const SPOTIFY_API_URL = "https://api.spotify.com/v1" as url
export const DEFAULT_CONTEXT_TITLE_URL = "http://no.data.found" as url
